### ISSUE TITLE
Result cannot return `non_subscription`

### ISSUE THREAD

**Issue by Adiis** [chargebee/chargebee-ruby#78]
According to the [documentation](https://apidocs.chargebee.com/docs/api/non_subscriptions), `Result` should be able to return `non_subscription`:

```ruby
require 'chargebee'
ChargeBee.configure(:site => "{site}",
  :api_key => "{site_api_key}")
result = ChargeBee::NonSubscription.process_receipt("cb-pjp7hcmrcbfmtjhle3smlwicu4",{
  :receipt => "Apple Based64 Encoded Receipt",
  :product => {
    :id => "app_store_charge_id",
    :price => "3399",
    :price_in_decimal => "33.99",
    :currency_code => "USD",
    :type => "CONSUMABLE"
    },
  :customer => {
    :id => "customer-123"
    }
  })
non_subscription = result.non_subscription
```

However, in [release 2.39.0](https://github.com/chargebee/chargebee-ruby/commit/65aee1d6ab52d5a9004442b0361c0e22a4f67c05#diff-3b9f8a218dd5d9fac43ccc6e40edd85e08c07db057f10e92590e5e302d9e8972L364-L367)  the `non_subscription()` method has been removed, which results in error:

```
NoMethodError: undefined method `non_subscription' for #<ChargeBee::Result:0x0000000109f9bff0...
```

**Comment by cb-alish** [comment]
hi @Adiis  this has been fixed in latest version v2.41.0. Please give it a try and get back if that doesn't work for you.

**Comment by Adiis** [comment]
Now it works as expected! Thank you, @cb-alish!
